bammetrics.wdl

```diff
@@ -88,5 +95,7 @@ workflow BamMetrics {
     }
 
     output {
+        File flagstats = Flagstat.flagstat
+        Array[File] picardMetricsFiles = picardMetrics.allStats
```

Comment:
Would it not be better to list the actual outputs?

Reply:
That is more than 20 different outputs...